static/src/app/library/library.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import leaflet from 'leaflet';
```

Comment:
One thing to keep in mind re: lint for unused imports - they don't actually act the same way as python imports do.  An import is shorthand for a node style `require`, meaning you want a resource available, but you don't always explicitly use it.   
`import Styles from '../style/sass/main.scss' `   
is another way of saying  
 `var Styles = require('../style/sass/main.scss')`  
and could really in this place just be  
`require('../style/sass/main.scss')`  
which just means that it will be available in the bundle and will affect the styling on the page.

Similarly, you can `require` a javascript file which has self-executing code if you need it for a page to work, and never explicitly call out to it as a library.

That's my understanding of it, at least.